### ISSUE TITLE
[OpenGL] Metal Max 2 Reloaded flashing black screens every now and then

### ISSUE THREAD

**Issue by TsukiZero** [melonDS-emu/melonDS#1569]
Switching to Software Rendering eliminates the issue, but this issue was not present in 0.9.4.

**Comment by TsukiZero** [comment]
Apparently there is something that 0.9.5 doesn't agree with that is in the 0.9.4 folder. Here's the 0.9.4 folder:
![image](https://user-images.githubusercontent.com/78157224/204073231-39369381-bf6d-4849-bf16-913d36d99d3e.png)

I got a suspicion those DLL files are the troublemakers.

**Comment by sandwichwater** [comment]
i get something similar that happens during dual screen segments #1537 , it also wasnt present in 0.9.4 but the builds right before that changed the opengl renderer broke it and it never got fixed for the final release of 0.9.5 . whatever it is , the opengl renderer definitely had something mess it up with the latest version that causes flashing in various places , this is the third instance ive seen flashing with the new opengl renderer

**Comment by FleetingFlame** [comment]
I just posted my own error about this, but I found that I got this glitch after AMD overhauled OpenGL in their drivers earlier this year.

**Comment by Jules-A** [comment]
> I just posted my own error about this, but I found that I got this glitch after AMD overhauled OpenGL in their drivers earlier this year.

So did you actually verify that it doesn't happen with driver versions 22.6.X or older because that wasn't stated in that issue you created?

**Comment by TsukiZero** [comment]
I have a NVIDIA GTX 970, so... ?

**Comment by FleetingFlame** [comment]
> So did you actually verify that it doesn't happen with driver versions 22.6.X or older because that wasn't stated in that issue you created?

Yes, I don't remember the driver version itself, but the issues only started after installing the OpenGl overhauled one (the one this year that was all over the news with 50%+ gains). I reinstalled the driver before it and the issue disappeared. Unfortunately, I had to reinstall windows between now and then so I don't have that driver :( .

**Comment by Jules-A** [comment]
> Unfortunately, I had to reinstall windows between now and then so I don't have that driver :( .
It's extremely simple to download an old driver version from AMD's site, they literally still list 22.5.1 as recommended driver... Considering OP has Nvidia I'd say either your issue is different or it's the same and nothing to do with drivers.
